collect_variables/scripts/download_stats.py

```diff
@@ -94,6 +94,34 @@
                         raise err
                     break
 
+            # JS
+            matches = re.finditer(r"npm (i|install)( -+.*)* (.*)",
+                                  str(readme))
+
+            for match in matches:
+                if name in match.group(3):
+                    print(f"Download stats for npm package '{name}'")
+                    try:
+                        if "@" in match.group(3):
+                            stats = requests.get(NPM_STATS.format(match.group(3)))
+                        else:
+                            stats = requests.get(NPM_STATS.format(name))
+                        print(stats.json()["downloads"])
+                        result.append({
+                            "repository_name":
+                            name,
+                            "owner":
+                            repo["owner"],
+                            "last_month":
+                            stats.json()["downloads"],
+                            "date":
+                            str(datetime.date.today())
+                        })
+
+                    except Exception as err:  # pylint: disable=broad-except
+                        print("Repository does not exist")
+                    break
+
         df_stats = pd.DataFrame(result)
```

Comment:
```suggestion
        df_stats = pd.DataFrame(result).sort_values(["owner", "repository_name"], inplace=True)
```

Might be good to sort the data to prevent a lot of changes when rerunning or updating the collection strategy.